bin/rucio

```diff
@@ -73,9 +81,14 @@ FAILURE = 1
 DEFAULT_SECURE_PORT = 443
 DEFAULT_PORT = 80
 
+# Setting up rich console.
+console = Console(theme=Theme(CLITheme.LOG_THEMES), soft_wrap=True)
```

Comment:
This setup, and the other setup in bin/rucio-admin, should probably be contained to a function in richclient and called within each function that uses the rich client to keep things more self-contained.